src/components/Profile.vue

```diff
@@ -36,8 +32,37 @@
 </template>
 
 <script>
+import axios from 'axios';
 export default {
-		
+    name:'Profile',
+    props:{},
+    data() {
+        return {
+            userData: {
+                username: '',
+                name:'pipi',
+                bio:'',
+                email:'',
+                geo: {
+                    lat:	'-37.3159',
+                    lng:	'81.1496'
+                },
+                radius: null,
+                interests: [],
+                expertise: undefined,
+                availability: null,
+            },
+        };
+    },
+    created () {
+        axios.get('https://my-json-server.typicode.com/manuelaguirre/my-json/users')
+            .then(response => this.displayUser(response.data));
+    },
+    methods: {
+        displayUser(user) {
+            this.userData = {...user};
+        }
```

Comment:
I think for now this is ok, because it will display the profile of a single user. The next step would be to have an api endpoint which returns a list of users, and have some component which renders a list with subcomponents for "user summary", and once one selects the user, it will have one screen with the profile details for the selected user. In that sense, I think we should think about having the user as a prop, or the url for the user as a prop, but I'm happy to leave that for later.